tensorboard/webapp/runs/views/runs_table/regex_edit_dialog_container.ts

```diff
@@ -121,9 +155,14 @@ export class RegexEditDialogContainer {
   constructor(
     private readonly store: Store<State>,
     public dialogRef: MatDialogRef<RegexEditDialogContainer>,
-    @Inject(MAT_DIALOG_DATA) data: {experimentIds: string[]}
+    @Inject(MAT_DIALOG_DATA)
+    data: {
+      experimentIds: string[];
+      expNameByExpId: Record<string, string>;
+    }
   ) {
     this.experimentIds = data.experimentIds;
+    this.expNameByExpId = data.expNameByExpId;
```

Comment:
This will be a bit annoying to change, but I think it's justified: currently we get expNameByExpId from the button container, pass it to the component, which passes to the dialog, which passes to the dialog container. But the dialog container has direct access to the store, so we can get expNameByExpId directly from a selector here without all the data passing.